packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -192,7 +196,6 @@ export default class RunPanelStore {
         observer.register('bot.contract', this.onBotContractEvent);
         observer.register('bot.contract', contract_card.onBotContractEvent);
         observer.register('bot.contract', transactions.onBotContractEvent);
-        observer.register('ui.log.success', journal.onLogSuccess);
```

Comment:
What's the reason removing this? Message from `scratch/utils` line 139 will not show if blocks loaded successfully.

Reply:
It's already registered in the `onMount` function.